src/core.c/Rakudo/Internals/PostcircumfixAdverbs.pm6

```diff
@@ -0,0 +1,509 @@
+      Array::Element::Access::exists-delete-any,
+      Array::Element::Access::exists-delete-kv-any,
+      Array::Element::Access::exists-delete-not-kv-any,
+      Array::Element::Access::exists-delete-p-any,
+      Array::Element::Access::exists-delete-not-p-any,
+      Array::Element::Access::not-exists-any,
+      Array::Element::Access::not-exists-kv-any,
+      Array::Element::Access::not-exists-not-kv-any,
+      Array::Element::Access::not-exists-p-any,
+      Array::Element::Access::not-exists-not-p-any,
+      Array::Element::Access::not-exists-delete-any,
+      Array::Element::Access::not-exists-delete-kv-any,
+      Array::Element::Access::not-exists-delete-not-kv-any,
+      Array::Element::Access::not-exists-delete-p-any,
+      Array::Element::Access::not-exists-delete-not-p-any,
+      Array::Element::Access::delete-any,
+      Array::Element::Access::delete-kv-any,
+      Array::Element::Access::delete-not-kv-any,
+      Array::Element::Access::delete-p-any,
+      Array::Element::Access::delete-not-p-any,
+      Array::Element::Access::delete-k-any,
+      Array::Element::Access::delete-not-k-any,
+      Array::Element::Access::delete-v-any
+    );
+
+    method ACCESS-ELEMENT-ANY-DISPATCH-CLASS(int $index) {
+        nqp::atpos($access-element-dispatch-any,$index)
+    }
+
+    # Constants to allow mapping of valid slice adverb combinations to
+    # a value that can be used in lookup table to get a dispatch table
+    # lookup value.
+    my constant SLICE_NO_ADVERBS = 0x0000;
+    my constant SLICE_DELETE     = 0x0001;  # :delete
+    my constant SLICE_EXISTS     = 0x0002;  # :exists
+    my constant SLICE_NOT_EXISTS = 0x0004;  # :!exists
+    my constant SLICE_KV         = 0x0008;  # :k
+    my constant SLICE_NOT_KV     = 0x0010;  # :!kv
+    my constant SLICE_P          = 0x0020;  # :p
+    my constant SLICE_NOT_P      = 0x0040;  # :!p
+    my constant SLICE_K          = 0x0080;  # :k
+    my constant SLICE_NOT_K      = 0x0100;  # :!k
+    my constant SLICE_V          = 0x0200;  # :v
+
+    # Array of above constants with their names, used for generating
+    # error messages.
+    my constant @pc-constant-name = (
+      SLICE_DELETE,     'delete',
+      SLICE_EXISTS,     'exists',
+      SLICE_NOT_EXISTS, '!exists',
+      SLICE_KV,         'kv',
+      SLICE_NOT_KV,     '!kv',
+      SLICE_P,          'p',
+      SLICE_NOT_P,      '!p',
+      SLICE_K,          'k',
+      SLICE_NOT_K,      '!k',
+      SLICE_V,          'v',
+    );
+
+    # Array that contains dispatch table value for postcircumfix
+    # adverbs.  When mapping into this array results in 0, then it
+    # was an unsupported combination of adverbs.  Any other value
+    # indicates an index value into a dispatch table that should
+    # be initialized in the order in which these combinations are
+    # initialized here.
+    my constant @pc-adverb-mapper = do {
+        my int $i = -1;
+        my uint16 @map;
```

Comment:
This seems to be the (only) place that make the JVM backend explode. If I just remove the ```uint16``` then rakudo compiles.

Reply:
... so it seems to be the same kind of problem as reported in https://github.com/rakudo/rakudo/issues/1666

Reply:
Hopefully addressed with https://github.com/rakudo/rakudo/commit/cbdc32bbc6

Reply:
I'm afraid, that won't help. It compiles with a non-native array (```my @map```). But probably that will cause problems in other places ...

I have no clue what goes wrong with native arrays on the JVM backend when compiling the core :(